boot/bootutil/src/loader.c

```diff
@@ -1270,7 +1270,7 @@ static inline void sec_slot_cleanup_if_unusable(void)
 }
 #endif /* defined(CONFIG_MCUBOOT_CLEANUP_UNUSABLE_SECONDARY) &&\
           defined(PM_S1_ADDRESS) || defined(CONFIG_SOC_NRF5340_CPUAPP) */
-
+bool owner_nsib[BOOT_IMAGE_NUMBER];
```

Comment:
should not be a global scope variable, plus define at top of file with other local variables